components/service/glean/README.md

```diff
@@ -119,14 +119,16 @@ for the command line switches used to pass the extra keys. These are the current
 |key|type|description|
 |---|----|-----------|
 | logPings | boolean (--ez) | If set to `true`, glean dumps pings to logcat; defaults to `false` |
-| sendPing | string (--es) | Sends the ping with the given name immediately. |
+| sendPing | string (--es) | Sends the ping with the given name immediately |
+| debugId | string (--es) | Tags all pings using the `X-Debug-ID` header combined with the given value which must match the pattern `[a-zA-Z0-9-]{1,20}` |
```

Comment:
For our users `X-Debug-ID` is not meaningful.
Can we change this to something like...
> Tags all outgoing data as debug pings, which will make them available for real-time validation. The value must ...